src/validators/radio.button.validator.ts

```diff
@@ -0,0 +1,24 @@
+import { logger } from "../utils/logger";
+
+export const isRadioButtonValid = (radioId: string): boolean => {
+
+  logger.debug("Check radio button is valid");
+
+  if (!radioId) {
+    logger.debug("No radio button id supplied");
+    return true;
+  }
+
+  const validRadioIds: string[] = ["yes", "no", "recently_filed"];
```

Comment:
you should use the RADIO_BUTTON_VALUES enum from utils/constants instead of the strings 'yes' 'no' 'recently_filed'